package.json

```diff
@@ -23,10 +23,11 @@
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start-client": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start": "npm run build && (cd server && npm start)"
```

Comment:
there is a great package to do this type of things
`https://www.npmjs.com/package/concurrently`